### ISSUE TITLE
Do not place comment on single line before closing brace but after code

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#328]
Most times, a comment describes something that is related to an operation.
That's why the comment should not be placed *after* the operation, but *before*. So the reader can easily associate the comment and the operation and does not need to guess if both are related or not.

Therefore, following code snippet should produce a warning:
```c#
try
{
    DoSomething();
}
catch (Exception ex)
{
    Debug.WriteLine(ex);

    // some comment
}
```

While following code snippet should **NOT** produce a warning:
```c#
try
{
    DoSomething();
}
catch (Exception ex)
{
    // some comment
    Debug.WriteLine(ex);
}
```

Similar here, following code should **NOT** produce a warning:
```c#
object SomeMethod()
{
    return DoSomething(); // some comment
}
```

**Comment by RalfKoban** [comment]
Additionally, following code should **NOT** produce a warning.

```c#
try
{
    DoSomething();
}
catch (Exception ex)
{
    // some comment
}
```
